src/Magento/FunctionalTestingFramework/Composer/Util/ComposerJsonFinder.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\FunctionalTestingFramework\Composer\Util;
+
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+
+/**
+ * Class ComposerJsonFinder searches composer json file for possible test module code paths
+ */
+class ComposerJsonFinder
```

Comment:
What do you think about moving both of these functions into `ComposerModuleResolver`? We can abstract out into a utility class when we find the need to use it outside of that context.

Reply:
They are actually utils and don't belong to composer or ComposerModuleResolver. but I am ok to move it to ComposerModuleResolver.